tfhe/src/integer/ciphertext/compressed_ciphertext_list.rs

```diff
@@ -155,187 +155,192 @@ mod tests {
     use super::*;
     use crate::integer::{gen_keys, IntegerKeyKind};
     use crate::shortint::parameters::list_compression::COMP_PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64;
-    use crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64;
+    use crate::shortint::parameters::{
+        PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64,
+        V1_0_PARAM_GPU_MULTI_BIT_GROUP_2_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64,
```

Comment:
This line made me confuse. We use the GPU parameter set to assert compatibility with compression, but we don't test the GPU backend here, right? In this case, probably the test in https://github.com/zama-ai/tfhe-rs/blob/4284a2e4bbe378ff95eae08f16923dacbd7d3e7d/tfhe/src/integer/gpu/ciphertext/compressed_ciphertext_list.rs#L496 should also be modified.